Kernel/Memory/MemoryManager.cpp

```diff
@@ -92,7 +93,9 @@ UNMAP_AFTER_INIT MemoryManager::MemoryManager()
 {
     s_the = this;
 
-    parse_memory_map();
+    if (auto parse_error = parse_memory_map(); parse_error.is_error())
+        PANIC("MM: failed to parse memory map: {}", parse_error.release_error());
```

Comment:
I am really not fond of this. If for some reason memory map parsing failed, I'd like it to fail on the exact line where it failed, which means we need to use `MUST()` - there's simple no reason to propagate an error back just to panic, which will make it much more harder to debug something in that part of the code.